### PR TITLE
chore(rspec) fix jenkins role unit tests - fixup of 3ac9b8b9

### PR DESCRIPTION
**Pull request by dduportal**
In the "heat of the battle", I forgot to push this commit in #2611

### CODE REVIEW

File: hieradata/rspec/role_jenkins.yaml [hunk 1]
```diff
@@ -0,0 +1,186 @@
+---
+lookup_options:
+  "^profile::jenkinscontroller::jcasc$":
+    merge:
+      strategy: deep
+      merge_hash_arrays: true
+
+profile::jenkinscontroller::ci_fqdn: 'jenkins.role.respec.test.local'
+# profile::jenkinscontroller::proxy_port: 443
```
Comment (lemeurherve) [inline hieradata/rspec/role_jenkins.yaml]: Why all these commented lines?

File: hieradata/rspec/profile_jenkinscontroller.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ lookup_options:
       strategy: deep
       merge_hash_arrays: true
 
-profile::jenkinscontroller::ci_fqdn: 'cert.ci.jenkins.io'
+profile::jenkinscontroller::ci_fqdn: 'jenkinscontroller.profile.respec.test.local'
```
Comment (lemeurherve) [inline hieradata/rspec/profile_jenkinscontroller.yaml]:
```suggestion
profile::jenkinscontroller::ci_fqdn: 'jenkinscontroller.profile.rspec.test.local'
```
Typo?

File: hieradata/rspec/role_jenkins.yaml [hunk 3]
```diff
@@ -0,0 +1,186 @@
+---
+lookup_options:
+  "^profile::jenkinscontroller::jcasc$":
+    merge:
+      strategy: deep
+      merge_hash_arrays: true
+
+profile::jenkinscontroller::ci_fqdn: 'jenkins.role.respec.test.local'
```
Comment (lemeurherve) [inline hieradata/rspec/role_jenkins.yaml]:
```suggestion
profile::jenkinscontroller::ci_fqdn: 'jenkins.role.rspec.test.local'
```
Same as above, typo?

File: hieradata/rspec/role_jenkins.yaml (hunk 1 above)
Comment (dduportal) [reply hieradata/rspec/role_jenkins.yaml]: it was me-PEBKAC .

Fixed :) Thanks for tha catch

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**